terra-batch-libchart/templates/_reverse-proxy.tpl

```diff
@@ -115,12 +115,10 @@ data:
         }
         {{ end }}
 
-        {{- if .Values.wds.enabled }}
         location /wds/ {
```

Comment:
We don't need to check if wds is enabled because it's always enabled/deployed, or why?